notebooks/work-in-progress/eia-epacems-crosswalk.ipynb

```diff
@@ -0,0 +1,299 @@
+{
```

Comment:
<p>I looked through the issue and my understanding is that you can access EPA information by using the <code>plant_id_eia</code>  (as shown by Karl's function pulling information from the FRS site) but there still isn't a clear mapping of <code>plant_id_eia</code>  to this "epa unit". This links at the top of the issue seemed promising, but the first link was dead and the second (all the eGrid files) seems like it would be out of date given that it stops in 2016. The issue also culminates with Greg talking about the file he made (that I'm mimicking here) so it seems like he would have incorporated that data if it pertained to direct id linkage. I can do some more digging though if you think there's more concrete information out there.</p>

Reply:
<p>On that note, however, it might be really useful to integrate the FRS Regional Ids into this same crosswalk table? I'm working on a function, but I can't figure out how to avoid using the notorious 'apply' function at the moment. Read: it's taking forever.</p>